src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$button-color: #fff;
+$blue-color: #00acdc;
+$text-grey-color: #616070;
+$color-big-text: #060b35;
+$button-width: 166px;
```

Comment:
```suggestion
$button-width: 100%;
```

Reply:
@volodymyr-soltys97 when I change 166px to 100%, the width breaks and the text stretches
![Uploading image.png…]()